RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -150,14 +153,15 @@
 
 # TRACK BUILDING
 import RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi
-detachedQuadStepTrajectoryBuilder = RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi.GroupedCkfTrajectoryBuilder.clone(
-    trajectoryFilter = dict(refToPSet_ = 'detachedQuadStepTrajectoryFilter'),
-    maxCand = 3,
-    alwaysUseInvalidHits = True,
-    estimator = 'detachedQuadStepChi2Est',
-    maxDPhiForLooperReconstruction = 2.0,
-    maxPtForLooperReconstruction = 0.7,
-)
+detachedQuadStepTrajectoryBuilder = RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi.GroupedCkfTrajectoryBuilder.clone()
+_detachedQuadStepTrajectoryBuilderParabolicMf = RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi.GroupedCkfTrajectoryBuilderParabolicMf.clone()
+trackingParabolicMf.toReplaceWith(detachedQuadStepTrajectoryBuilder,_detachedQuadStepTrajectoryBuilderParabolicMf.clone())
+detachedQuadStepTrajectoryBuilder.trajectoryFilter = dict(refToPSet_ = 'detachedQuadStepTrajectoryFilter')
+detachedQuadStepTrajectoryBuilder.maxCand = 3
```

Comment:
it seems more practical to do similar to fitting and define smth like `GroupedCkfTrajectoryBuilderIterativeDefault_cfi` and simply replace the MF parts there.
The PR line changes will reduce by an order of magnitude